oauth2client/client.py

```diff
@@ -1862,7 +1863,7 @@ def FromResponse(cls, response):
         })
         if 'expires_in' in response:
             kwargs['user_code_expiry'] = (
-                datetime.datetime.now() +
+                _NOW() +
```

Comment:
We would break [this code](http://furyu.hatenablog.com/entry/20150820/1440064896). But that's the only code I could find.

I agree that the value should be UTC. Maybe make the change in its own pull request, so it at least shows up prominently in release notes?